status/flowcells.py

```diff
@@ -31,17 +32,26 @@ def formatDate(date):
 
 class FlowcellsHandler(SafeHandler):
     """ Serves a page which lists all flowcells with some brief info.
+    By default shows only flowcells form the last 6 months, use the parameter
+    'all' to show all flowcells.
     """
-    def list_flowcells(self):
+    def list_flowcells(self, all=False):
         flowcells = OrderedDict()
         temp_flowcells={}
-        fc_view = self.application.flowcells_db.view("info/summary",
-                                                     descending=True)
-        for row in fc_view:
-            temp_flowcells[row.key] = row.value
-
-        xfc_view = self.application.x_flowcells_db.view("info/summary",
-                                                     descending=True)
+        if all: # fc_view are from 2016 and older
+            fc_view = self.application.flowcells_db.view("info/summary",
+                                                        descending=True)
+            for row in fc_view:
+                temp_flowcells[row.key] = row.value
+
+        if all:
```

Comment:
This if could be merged with the above if since its the same condition, couldn't it? That leaves us with a single if-else.